tic-tac-toe-test/BoardTest.cs

```diff
@@ -64,15 +62,39 @@ public void IsFullWhenAllSlotsAreFilled()
         [Fact]
         public void IsNotFullWhenThereIsAtLeastOneEmptySlot()
         {
-            List<char> fullBoardState = new List<char> {
+            List<char> notFullBoardState = new List<char> {
                 'X', 'O', 'X',
                 'O', 'X', 'O',
                 'X', 'O', '_',
                 };
 
-            Board board = new Board(fullBoardState);
+            Board board = new Board(notFullBoardState);
 
             Assert.False(board.IsFull());
         }
+
+        [Fact]
+        public void ReturnsAllRowsColumnsAndDiagonals()
+        {
+            List<char> gameState = new List<char> {
+                'X', 'X', 'X',
+                'O', '_', 'O',
+                '_', 'O', 'X',
+                };
+
+            Board board = new Board(gameState);
+
+            List<List<char>> expectedLines = new List<List<char>>();
+            expectedLines.Add(new List<char> { 'X', 'X', 'X' });
+            expectedLines.Add(new List<char> { 'O', '_', 'O' });
+            expectedLines.Add(new List<char> { '_', 'O', 'X' });
+            expectedLines.Add(new List<char> { 'X', 'O', '_' });
+            expectedLines.Add(new List<char> { 'X', '_', 'O' });
```

Comment:
Is this how you create a List of Lists? Is there any easier way to add lots of lists of chars to a list?!

Reply:
Are you talking about https://github.com/RabeaGleissner/dotnet-tic-tac-toe/pull/4/files#diff-19f9eac83705df9759232fa3a8e087d1dd636c9a122afa5827609dffaa07273dR87 ? I'm also learning and i think that's the idiomatic way of doing it, from my googling.

Reply:
Yes... I was hoping that there was some way to initialise the whole list of lists with content, rather than initialising the empty list of lists and then having to add each child list separately with the `Add()` function. Did a bit of googling but couldn't find anything useful, but I just can't fathom that this is the most efficient way! 😂